App/Components/Interaction/CompFriendList.razor

```diff
@@ -0,0 +1,94 @@
+@inherits UserViewComponentBase
+@implements IInteractionSubscriber
+
+@using Bamboozlers.Classes.Utility.Observer
+@using Bamboozlers.Classes.AppDbContext
+@using Bamboozlers.Classes.Data
+@using Bamboozlers.Classes.Services.UserServices
+@using Bamboozlers.Components.Utility
+
+<div style="margin-top: 10px; width: 100%">
+    <Heading>
+        @if (Friends.Count > 0)
+        {
+            <Badge ElementId="BlockCountBadge"
+                   Color="Color.Dark">
+                @Friends.Count
+            </Badge>
+        }
+        Friends
+    </Heading>
+    <Divider/>
+    @if (Friends.Count > 0)
+    {
+        <div class="scrollbar-container" style="display: inline-block; position: relative; left: 12%; width: 75%; max-height: 100%; margin-left: 10px; margin-right: 10px;">
+            <div>
+                
+            </div>
+            @foreach (var user in Friends)
+            {
+                @GetFriendFragment(user)
+                ;
+            }
+        </div>
+    }
+    else
+    {
+        <p>Nothing to see here...</p>
+        <p>You haven't made any friends.</p>
+    }
+</div>
+
+@code {
+    [Inject] private IUserInteractionService UserInteractionService { get; set; } = default!;
+    
+    [CascadingParameter] private EventCallback<OpenChatArgs> OpenChatCallback { get; set; }
+    [CascadingParameter] private EventCallback ClosePopupCallback { get; set; }
+    
+    [Parameter] public bool IsPopup { get; set; } = false;
+    
+    private User Self { get; set; } = default!;
+    private List<User> Friends { get; set; } = [];
+
+    protected override async Task OnInitializedAsync()
+    {
+        Self = (await AuthService.GetUser())!;
+        UserInteractionService.AddSubscriber(this);
+        await GetFriendData();
+    }
+
+    public RenderFragment GetFriendFragment(User user)
```

Comment:
I redesigned this in my branch to look like:

https://github.com/Bawnorton/Bamboozlers/blob/19beb6d771603bf7924ae6edffa722b2a60dfc8c/App/Components/Friend/CurrentFriends.razor#L45-L56

Which I believe looks better. The Message friend button being readily available is also a good idea.

Reply:
https://github.com/Bawnorton/Bamboozlers/assets/18416784/f899983a-e127-48b9-9f55-3a1284d2e12f

Reply:
will swap what i've got out for this